src/css/profile/wearable/changeable/theme-circle/checkboxradio.less

```diff
@@ -139,7 +140,7 @@ input[type="radio"] {
 		}
 	}
 	&:disabled {
-		background: radial-gradient(transparent, transparent 59%, @color_checkboxradio_radiobox_line_off_dim 63%, @color_checkboxradio_radiobox_line_off_dim 68%, transparent 72%, transparent);
+		background: radial-gradient(transparent, transparent 63%, @color_checkboxradio_radiobox_line_off_dim 65%, @color_checkboxradio_radiobox_line_off_dim 68%, transparent 72%, transparent);
```

Comment:
Is it right  @color_checkboxradio_radiobox_line_off_dim ?
It was defined by @color_checkboxradio_radiobox_line_off: fade(@controller-off-color, 80%); in theme,changeable.less.
It looks different value in GUI Guide.

Reply:
I verified color also. In my opinion color of disabled radio off should be #8f8f8f / 24% - current value is #8f8f8f / 30%

Reply:
I verified again. There is correct color.

Reply:
:) . I have to correct my comment again. Valid color for disabled radio off is: #8f8f8f / 30% according to html guideline.

Reply:
Thanks for your checking ^^.